service/host.go

```diff
@@ -183,8 +183,8 @@ func (s *host) shutdown(err error, reason string, exitCode *int) (bool, error) {
 	return true, err
 }
 
-// AddModules adds the given modules to a service host
-func (s *host) AddModules(modules ...ModuleCreateFunc) error {
+// addModules adds the given modules to a service host
```

Comment:
I prefer to err on the side of having comments. Let's not remove comments because they are not required by linting :)